modules/base/scale/staticscale.cpp

```diff
@@ -58,7 +58,7 @@ double StaticScale::scaleValue(const UpdateData&) const {
     return _scaleValue;
 }
 
-StaticScale::StaticScale() : _scaleValue(ScaleInfo, 1.0, 1.0, 1e6) {
+StaticScale::StaticScale() : _scaleValue(ScaleInfo, 1.0, 0.1, 100) {
```

Comment:
This is kind of funny - I keep changing this locally depending on what I want to do for the moment.
At some point it would be nice to have a straight-forward way to set these limits more dynamically based on what kind of object the scale applies to. Now's not the time though :) I think the factor 100 is probably at least as good as 1e6, so I'm fine with using 100 for now. :)

Reply:
Ok, you are right. I changed it because I figured it was impossible to use the slider for objects such as spacecrafts, planets etc. but I can imagine stars and other far away objects would need a very large scale when testing.